warehouse/static/sass/components/_sidebar-section.scss

```diff
@@ -6,6 +6,15 @@
   h3 {
     font-size: 18px;
   }
+
+  dt {
+    font-weight: bold;
+    margin-top: 0.3em;
+  }
+
+  dd {
+    padding-left: 1em;
+  }
 }
```

Comment:
It would be better to make this CSS generic by moving it into base/_lists.scss :)